src/js/components/List/stories/Order.js

```diff
@@ -4,9 +4,14 @@ import { Grommet, Box, List } from 'grommet';
 import { grommet } from 'grommet/themes';
 
 const locations = [
-  'Boise',
-  'Fort Collins',
-  'Los Gatos',
+  `Lorem ipsum dolor sit amet, consectetur adipiscing elit,
```

Comment:
I'm not seeing the new prop being tested here.

Reply:
just added sorry forgot to push that part up